test/unit/org/apache/cassandra/metrics/BufferPoolMetricsTest.java

```diff
@@ -18,25 +18,40 @@
 
 package org.apache.cassandra.metrics;
 
+import java.util.Arrays;
 import java.util.Random;
 
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.exceptions.ConfigurationException;
 import org.apache.cassandra.io.compress.BufferType;
 import org.apache.cassandra.utils.memory.BufferPool;
 
+import static org.apache.cassandra.config.CassandraRelevantProperties.USE_MICROMETER;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.junit.Assert.assertEquals;
 
+@RunWith(Parameterized.class)
 public class BufferPoolMetricsTest
 {
     private BufferPool bufferPool;
     private BufferPoolMetrics metrics;
+    
+    // Test with both MicrometerBufferPoolMetrics and CodehaleBufferPoolMetrics
+    @Parameterized.Parameters(name = "useMicroMeterMetrics {0}")
+    public static Iterable<Object[]> params()
+    {
+        return Arrays.asList(new Object[][] { { false}, { true} });
```

Comment:
```suggestion
        return Arrays.asList(new Object[][] { {false}, {true} });
```

Reply:
@sbtourist Are you sure your code formatting suggestion follows OSS style guide? Note CC follows OSS style guide, which is very different from CNDB.
When I apply C* style, it confirms Dan's code.
Note `UnifiedCompactionStrategyTest` uses incorrect formatting to my understanding.

Reply:
I didn't double check tbh. It looks odd, but sure feel free to ignore my suggestion if that is in line with the OSS formatter.

Reply:
Hmm, I reformatted (C* formatting) and it added spaces like this:
```
        return Arrays.asList(new Object[][]{ { false }, { true } });
```

Agree that the original with a space on just one side was wrong/odd.